.luacheckrc

```diff
@@ -0,0 +1,657 @@
+                get_metadata = {},
+                get_name = {},
+                get_short_description = {},
+                get_stack_max = {},
+                get_tool_capabilities = {},
+                get_wear = {},
+                is_empty = {},
+                is_known = {},
+                item_fits = {},
+                peek_item = {},
+                replace = {},
+                set_count = {},
+                set_metadata = {},
+                set_name = {},
+                set_wear = {},
+                take_item = {},
+                to_string = {},
+                to_table = {},
+		    },
+		},
+        PerlinNoise = {
+            fields = {
+                get_2d = {},
+                get_3d = {},
+            },
+        },
+        PerlinNoiseMap = {
+            fields = {
+                calc_2d_map = {},
+                calc_3d_map = {},
+                get_2d_map = {},
+                get_2d_map_flat = {},
+                get_3d_map = {},
+                get_3d_map_flat = {},
+                get_map_slice = {},
+            },
+        },
+		PseudoRandom = {
+		    fields = {
+		        next = {},
+		    },
+		},
+		PcgRandom = {
+		    fields = {
+		        next = {},
+		        rand_normal_dist = {},
+		    },
+		},
+		SecureRandom = {
+		    fields = {
+		        next_bytes = {},
+		    },
+		},
+		Settings = {
+		    fields = {
+                get = {},
+                get_bool = {},
+                get_flags = {},
+                get_names = {},
+                get_np_group = {},
+                remove = {},
+                set = {},
+                set_bool = {},
+                set_np_group = {},
+                to_table = {},
+                write = {},
+		    },
+		},
+        VoxelArea = {
+            fields = {
+                MaxEdge = {},
+                MinEdge = {},
+                contains = {},
+                containsi = {},
+                containsp = {},
+                getExtent = {},
+                getVolume = {},
+                index = {},
+                indexp = {},
+                iter = {},
+                iterp = {},
+                new = {},
+                position = {},
+                ystride = {},
+                zstride = {},
+            },
+        },
+        VoxelManip = {
+            fields = {
+                calc_lighting = {},
+                get_data = {},
+                get_emerged_area = {},
+                get_light_data = {},
+                get_node_at = {},
+                get_param2_data = {},
+                read_from_map = {},
+                set_data = {},
+                set_light_data = {},
+                set_lighting = {},
+                set_node_at = {},
+                set_param2_data = {},
+                update_liquids = {},
+                update_map = {},
+                was_modified = {},
+                write_to_map = {},
+            },
+        },
+
+        minetest = {
```

Comment:
This is too verbose and is outdated very quickly when new API is added to Minetest. It's a good idea to check for valid Minetest calls but I don't think it's worth it for this relatively small mod.